hbase-common/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupInfo.java

```diff
@@ -114,6 +120,14 @@ public boolean removeServer(Address hostPort) {
     return servers.remove(hostPort);
   }
 
+  public Map<String, String> getConfiguration() {
+    return configuration;
+  }
+
+  public void setConfiguration(Map<String, String> configuration) {
+    this.configuration = configuration;
+  }
+
```

Comment:
`TableDescriptor` and `ColumnFamiltyDescriptor` have a similar requirement, but provide a richer interface for interacting with their maps. Consider providing similar?

Reply:
You're right, it should be replaced with richer interface.I'll do it.